nmigen/hdl/ast.py

```diff
@@ -254,6 +254,38 @@ def __repr__(self):
 C = Const  # shorthand
 
 
+class AnyValue(Value):
+    def __init__(self, shape):
+        super().__init__(src_loc_at=1)
+        if isinstance(shape, int):
+            shape = shape, False
+        self.nbits, self.signed = shape
+        if not isinstance(self.nbits, int) or self.nbits < 0:
+            raise TypeError("Width must be a non-negative integer, not '{!r}'", self.nbits)
+
+    def shape(self):
+        return self.nbits, self.signed
+
+    def _rhs_signals(self):
+        return ValueSet()
+
+
+class AnyConst(AnyValue):
+    def __init__(self, shape):
```

Comment:
This is a no-op--this class does not need the `__init__` method. (Though, once you remove it, you should change `src_loc_at` above.)